### PR TITLE
Fixes incorrect logic in determining ImageVersion from LDFlags variable

### PR DESCRIPTION
**Pull request by phantomjinx**

### CODE REVIEW

**Review by tadayosi — CHANGES_REQUESTED** [review]
You should merge it into `main` branch, not `1.x`. The same applies to other changes you've made so far.

**Comment by tadayosi** [comment]
Please don't use `1.x` until we want to start 2.0 development.

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,21 @@
 FROM golang:1.20-alpine3.18 AS builder
 
+ARG HAWTIO_ONLINE_VERSION=latest
+ARG HAWTIO_ONLINE_IMAGE_NAME=docker.io/hawtio/hawtio
```
Comment (tadayosi) [inline Dockerfile:4]: Let's move to `quay.io`. See #65

**Comment by phantomjinx** [comment]
> You should merge it into `main` branch, not `1.x`. The same applies to other changes you've made so far.

Yes, an error, thx.

Just FYI, the 1.x branch is being used to do the downstream build so once the changes are in `main`, they are being rebased over to it.

File: Dockerfile (hunk 1 above)
Comment (phantomjinx) [reply Dockerfile:4]: > Let's move to `quay.io`. See #65

This will be done in a separate PR to keep the concerns separate.

**Comment by phantomjinx** [comment]
Incorrect branch being merged to.

New PRs at:
- https://github.com/hawtio/hawtio-operator/pull/86
- https://github.com/hawtio/hawtio-operator/pull/87
- https://github.com/hawtio/hawtio-operator/pull/85

**Pull request closed without merging**